src/components/Disturber.tsx

```diff
@@ -1,15 +1,17 @@
 import { DrawerNavigationProp } from '@react-navigation/drawer';
 import React, { useEffect, useState } from 'react';
-import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { FlatList, ScrollView, StyleSheet, TouchableOpacity, View } from 'react-native';
 
-import { Icon, colors, normalize } from '../config';
-import { addToStore, findClosestItem, isActive, readFromStore } from '../helpers';
+import { Icon, colors, device, normalize } from '../config';
+import { addToStore, findClosestItem, imageWidth, isActive, readFromStore } from '../helpers';
 import { useHomeRefresh, useStaticContent } from '../hooks';
 
 import { Button } from './Button';
 import { Image } from './Image';
 import { BoldText, RegularText } from './Text';
 import { Wrapper, WrapperHorizontal } from './Wrapper';
+import { ImagesCarousel } from './ImagesCarousel';
+import Carousel from 'react-native-snap-carousel';
```

Comment:
organize imports (manually?) to get this up.
or...is this even used?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/972/commits/fb9727fe6e376ed39e7259b0d68dd15527dd1fbc